salt/top.sls

```diff
@@ -156,6 +156,12 @@ base:
     - fluentd.config
   'purpose:sandbox':
     - match: grain
+    - edx.prod
+    - edx.run_ansible
+    - edx.patch_nginx
+    - edx.edxapp_global_pre_commit
+    - edx.etc_hosts
```

Comment:
We don't need this for sandbox as it is a hack for fixing issues with xqueue latency